npc/re/merchants/mysterious_cookie_shop.txt

```diff
@@ -0,0 +1,100 @@
+//===== rAthena Script ======================================= 
+//= Mysterious Cookie Shop
+//===== Changelogs: ==========================================
+//= 1.0 Initial version from replays. [Capuche]
+//============================================================
+
+malangdo,124,152,3	script	Mysterious Cookie Shop#stone	4_COOKIEHOUSE,{
+	mes "[Mysterious Cookie Shop]";
+	mes "" + mesitemlink(7000) + " can be exchanged for anything you crave.";
+	next;
+	switch( select( "^4d4dffTop^000000 Stones", "^1C8008Middle^000000 Stones", "^ff0000Lower^000000 Stones", "^9905d8Garment^000000 Stones", "^8B4513Dual and effect^000000 Stones", "^9905d8Costumes^000000 on the ��clo��the��sline��" ) ) {
```

Comment:
Is this only an issue with github again or are there some fancy unicode characters in the last menu item?

Reply:
It's using special characters
![Capture d’écran (1543)](https://github.com/rathena/rathena/assets/5231893/3c97e3e8-985b-4c61-bb89-d0032fd6dde5)